release_images.yml

```diff
@@ -2,23 +2,23 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "tensorflow"
-    version: "2.8.0"
+    framework: "autogluon"
+    version: "0.3.2"
     arch_type: "x86"
     customer_type: "sagemaker"
     training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py39"]
-      os_version: "ubuntu20.04"
+      python_versions: ["py38"]
+      os_version: "ubuntu18.04"
       cuda_version: "cu112"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
       # to images being published.
       force_release: False  # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
     inference:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py39" ]
+      device_types: [ "cpu" ]
```

Comment:
Only CPU inference needs to be released?

Reply:
Correct. There is only a CPU dockerfile and image for AG inference -- https://github.com/aws/deep-learning-containers/tree/master/autogluon/inference/docker/0.3.2/py3